Resources/Locale/en-US/_NF/guidebook/guides.ftl

```diff
@@ -12,4 +12,6 @@ guide-entry-shipyard-legman = Legman
 guide-entry-shipyard-liquidator = Liquidator
 guide-entry-shipyard-pioneer = Pioneer
 guide-entry-shipyard-searchlight = Searchlight
-guide-entry-shipyard-spirit = Spirit
\ No newline at end of file
+guide-entry-shipyard-spirit = Spirit
+guide-entry-piloting = Piloting
+guide-entry-hiring = Hiring Crew
```

Comment:
Recommendation: group these together under guide-entry-bank, leave a space between Shipyard and the shipyard entries (Ambition and friends).  Leave an empty line at EOF.

Reply:
Done